dspace/config/registries/openaire4-relationships.xml

```diff
@@ -0,0 +1,91 @@
+<relationships>
+
+	<!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_creator.html 
+		The relation between a Publication and a Person regarding the authorship -->
+	<type>
+		<leftType>Publication</leftType>
+		<rightType>Person</rightType>
+		<leftwardType>isAuthorOfPublication</leftwardType>
+		<rightwardType>isPublicationOfAuthor</rightwardType>
+		<leftCardinality>
+			<min>0</min>
+		</leftCardinality>
+		<rightCardinality>
+			<min>0</min>
+		</rightCardinality>
+	</type>
+	<!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_creator.html 
+		The relation between a Publication and an Organization regarding the authorship -->
+	<type>
+		<leftType>Publication</leftType>
+		<rightType>Organization</rightType>
+		<leftwardType>isAuthorOfPublication</leftwardType>
+		<rightwardType>isPublicationOfAuthor</rightwardType>
+		<leftCardinality>
+			<min>0</min>
+		</leftCardinality>
+		<rightCardinality>
+			<min>0</min>
+		</rightCardinality>
+	</type>
+	<!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_contributor.html 
+		The relation between a Publication and a Person regarding the contribution -->
+	<type>
+		<leftType>Publication</leftType>
+		<rightType>Person</rightType>
+		<leftwardType>isContributorOfPublication</leftwardType>
+		<rightwardType>isPublicationOfContributor</rightwardType>
+		<leftCardinality>
+			<min>0</min>
+		</leftCardinality>
+		<rightCardinality>
+			<min>0</min>
+		</rightCardinality>
+	</type>
+	<!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_contributor.html 
+		The relation between a Publication and an Organization regarding the contribution -->
+	<type>
+		<leftType>Publication</leftType>
+		<rightType>Organization</rightType>
+		<leftwardType>isContributorOfPublication</leftwardType>
+		<rightwardType>isPublicationOfContributor</rightwardType>
+		<leftCardinality>
+			<min>0</min>
+		</leftCardinality>
+		<rightCardinality>
+			<min>0</min>
+		</rightCardinality>
+	</type>
+	<!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_projectid.html 
+		The relation between a Publication and an Organization regarding the Funder -->
+	<type>
+		<leftType>Publication</leftType>
+		<rightType>Project</rightType>
+		<leftwardType>isProjectOfPublication</leftwardType>
+		<rightwardType>isPublicationOfProject</rightwardType>
+		<leftCardinality>
+			<min>0</min>
+		</leftCardinality>
+		<rightCardinality>
+			<min>0</min>
+		</rightCardinality>
+	</type>
+	<!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_projectid.html 
+		The relation between a Funded Project and an Organization regarding the Funder -->
+	<type>
+		<leftType>Project</leftType>
+		<rightType>Organization</rightType>
+		<leftwardType>isFundingAgencyOfProject</leftwardType>
```

Comment:
Should it be better <leftwardType>isFundingStreamOfProject</leftwardType>

Reply:
I've discuss this suggestion with @josekarvalho and we agree that Funding Stream is applied to a different context. In this case we need to relate a project with a funding agency like: World Bank, European Commission, our National FCT, ... In a particular example: the funding stream would be "Asylum and Migration Fund (AMIF)" of the European Commission (EC).
You have this example of what it intend to output:
https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_projectid.html
```
<oaire:fundingReferences>
 <oaire:fundingReference>
  <oaire:funderName>European Commission</datacite:funderName>
  <oaire:funderIdentifier funderIdentifierType="Crossref Funder ID">http://doi.org/10.13039/100010661</oaire:funderIdentifier>
  <oaire:fundingStream>Horizon 2020 Framework Programme</oaire:fundingStream>
  <oaire:awardNumber awardURI="http://cordis.europa.eu/project/rcn/194062_en.html">643410</oaire:awardNumber>
  <oaire:awardTitle>Open Access Infrastructure for Research in Europe 2020</oaire:awardTitle>
 </oaire:fundingReference>
</oaire:fundingReferences>
```